ansible_collections/arista/avd/roles/eos_designs/templates/designs/l3ls-evpn/logic/l2leaf.j2

```diff
@@ -7,6 +7,8 @@
 {%             set switch.mgmt_ip = l2leaf.node_groups[l2leaf_node_group].nodes[node].mgmt_ip %}
 {%             set switch.spanning_tree_mode = l2leaf.node_groups[l2leaf_node_group].spanning_tree_mode | arista.avd.default(
                                                l2leaf.defaults.spanning_tree_mode ) %}
+{%             set switch.spanning_tree_root_super = l2leaf.node_groups[l2leaf_node_group].spanning_tree_root_super | arista.avd.default(
```

Comment:
Would a L2 leaf ever be "Root Super" ? I thought it was only relevant for EVPN Leaf?